exercises/calculator.cpp

```diff
@@ -10,3 +10,39 @@
   Multiplication: 8
   Division: 2
 */
+
+double sum(double a, double b){
+  return a+b;
+}
+
+double difference(double a, double b){
+  return a-b;
+}
+
+double moltiplication(double a, double b){
+  return a*b;
+}
+
+double division(double a, double b){
+  return a/b;
+}
+
+#include <iostream>
+using namespace std;
+
+int main(){
+  double x, y;
+
+  cout<<"Insert first number: ";
+  cin>>x;
+
+  cout<<"Insert second number: ";
+  cin>>y;
+
+  cout<<"SUM: "<<sum(x, y)<<endl;
+  cout<<"Difference: "<<difference(x, y)<<endl;
+  cout<<"Moltiplication: "<<moltiplication(x, y)<<endl;
+  cout<<"Division: "<<division(x, y)<<endl;
+
+  return 0;
```

Comment:
Meglio usare una macro per i valori handcoded , come lo 0 nel return.
 Questo perchè le macro create con #define non una variabile perchè occupa memora, il simbolo di macro viene sostituito con il valore reale in fase di compilazione, per info: https://www.arduino.cc/reference/en/language/structure/further-syntax/define/:
```diff
+ #define EXIT_SUCCESS 0
....
+ return EXIT_SUCCESS;
- return 0;
```